Release-Notes.adoc

```diff
@@ -1,74 +1,82 @@
-* A `helm upgrade` command from 1.0 to 1.0.1 (scf 2.6.11 to 2.7.0) requires the use of `--force` to drop an unnecessary persistent volume. Note that `helm upgrade` only works for multi-node clusters when running with a proper HA storage class (e.g. `hostpath` will not work as required stateful data can be lost). 
-
-== 1.1 Release April, 2018
-
-* Support for Azure AKS 
-* Backup/restore cf CLI plugin 
-* Helm charts for PostgreSQL and MySQL service broker sidecars 
-* Ability to specify multiple external IP addresses (please see Known Issues below on impact to upgrades)  
-* MySQL now a clustered role 
-* Mysql-proxy enabled for UAA 
-* UAA has more logging enabled, so `SCF_LOG_HOST`, `SCF_LOG_PORT` and `SCF_LOG_PROTOCOL` have been exposed 
-* TCP routing ports are configurable and can be templatized 
-* CPU limits can be set 
-* Kubernetes annotations enabled so operators can specify what nodes particular roles can be run on 
-* Fixed cloud controller clock so that it will wait until API is ready 
-* Overhauled how secret rotation works regarding to upgrades 
-* Pods no longer random fail to start occasionally 
-* Includes these CF component versions 
-** diego-release: 1.35 
-** cf-mysql-release: 36.10.0 
-** cflinuxfs2-release: 1.187.0 
-** routing-release: 0.172.0 
-** garden-runc-release: 1.11.1 
-** nats-release: 22 
-** capi-release: 1.49.0 
-* Includes these CF buildpack versions 
-** go-buildpack-release: 1.7.19-16-g37cc6b4 
-** binary-buildpack-release: 1.0.17 
-** nodejs-buildpack-release: 1.5.30-13-g584d686 
-** ruby-buildpack-release: 9adff61 
-** php-buildpack-release: ea8acd0 
-** python-buildpack-release: 1.5.16-14-ga2bbb4c 
-** staticfile-buildpack-release: 1.4.0-12-gdfc6c09 
-** dotnet-core-buildpack-release: 1.0.26-14-gf951834 
-** java-buildpack-release: 3.16-18-gfeab2b6 
+= Release Notes - SUSE Cloud Application Platform
+
+== 1.1.0 Release April 2018
+
+=== What's new 
+
+* Now supported on Microsoft Azure Container Services (AKS)
+* Cloud Foundry component and buildpack updates (see <<features-and-fixes,Features and Fixes>>)
+* PostgreSQL and MySQL service broker sidecars, configured and deployed via Helm
+* +cf backup+ CLI plugin for saving, restoring, or migrating CF data and applications 
+
+See the link:https://www.suse.com/documentation/cloud-application-platform-1/[Deployment Guide] for details information how to deploy Cloud Application Platform.
+
+=== Configuration Changes
+
+IMPORTANT: Changes to the format of `values.yaml` for SCF and UAA require special handling when upgrading from 1.0 to 1.1 if you are reusing configuration files (e.g. `scf-config-values.yaml`):
+
+** Several values formerly set under `env:` are now set under `secrets:`. Any `_PASSWORD`, `_SECRET`, `_CERT`, or `_KEY` value explicitly set in `values.yaml` for CAP 1.0 should be moved into the `secrets:` section before running `helm upgrade` with the revised `values.yaml` 
```

Comment:
All secrets follow this rule.
I wonder if we should make a copy of https://github.com/SUSE/kubernetes-charts-suse-com/blob/master/stable/cf/values.yaml next to this document and reference it.
IMO it would be _very_ useful.